app/helpers/middleware_deployment_helper/textual_summary.rb

```diff
@@ -23,4 +23,8 @@ def textual_name
   def textual_nativeid
     @record.nativeid
   end
+
+  def textual_enabled
+    {:label => 'Enabled/Disabled', :value => @record.enabled ? 'Enabled' : 'Disabled'}
```

Comment:
iirc the label should be with translation - see other similar files for examples. it has something to do with _(